Revit_Core_Engine/Query/DuctSectionProfile.cs

```diff
@@ -36,41 +36,34 @@ public static partial class Query
         /***************************************************/
         /****               Public Methods              ****/
         /***************************************************/
-
-        [Description("Section profile of round and rectangular ducts converted from a Revit duct.")]
-        [Input("revitDuct", "Revit duct to be converted into a section profile.")]
-        [Input("settings", "Revit settings.")]
-        [Input("refObjects", "Referenced objects.")]
-        [Output("profile", "Converted BHoM duct section profile converted.")]
+        
+        [Description("Query a Revit duct to extract a BHoM duct section profile.")]
+        [Input("revitDuct", "Revit duct to be converted into a BHoM section profile.")]
+        [Input("settings", "Revit adapter settings.")]
+        [Input("refObjects", "A collection of objects processed in the current adapter action, stored to avoid processing the same object more than once.")]
+        [Output("profile", "BHoM section profile for a duct extracted from a Revit duct.")]
         public static BH.oM.MEP.SectionProperties.SectionProfile DuctSectionProfile(this Autodesk.Revit.DB.Mechanical.Duct revitDuct, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
         {
             settings = settings.DefaultIfNull();
 
             IProfile profile = revitDuct.Profile(settings, refObjects);
 
             // Lining thickness
-            double liningThickness = revitDuct.LookupParameterDouble(BuiltInParameter.RBS_LINING_THICKNESS);
+            //double liningThickness = revitDuct.LookupParameterDouble("Lining Thickness");
+            double liningThickness = revitDuct.LookupParameterDouble(BuiltInParameter.RBS_REFERENCE_LINING_THICKNESS);
```

Comment:
This functions as expected and am happy with the functionality. Question for @pawelbaran prior to approval if this is an acceptable practice as this was my earlier suggestion to use the RBS_REFERENCE params.

Reply:
Yeah, I think it is fine, as long as the correct parameter value is extracted - if you know which parameter value you want to extract from Revit, you can check its underlying `BuiltinParameter` value using Revit Lookup. That should be the ultimate answer to your question 😉 

By the way, could you please make sure that no commented code has been left behind @AndreBalasian?

Reply:
Thanks @pawelbaran and @AndreBalasian seems to be functioning as expected with the latest changes. I'm happy to consider my specific query resolved. Please mark resolved once the commented code has been removed as requested. 😄